<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qProgArmDialog</class>
 <widget class="QDialog" name="qProgArmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>422</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qProgArmDialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qProg_arm1986.qrc">
    <normaloff>:/new/prefix1/images/mainwin.png</normaloff>:/new/prefix1/images/mainwin.png</iconset>
  </property>
  <widget class="QComboBox" name="comboBox_File">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Browse">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>46</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
   <property name="icon">
    <iconset resource="qProg_arm1986.qrc">
     <normaloff>:/new/prefix1/images/open.bmp</normaloff>:/new/prefix1/images/open.bmp</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_WriteDevice">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Program device</string>
   </property>
   <property name="icon">
    <iconset resource="qProg_arm1986.qrc">
     <normaloff>:/new/prefix1/images/program.png</normaloff>:/new/prefix1/images/program.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Verify">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Verify device</string>
   </property>
   <property name="icon">
    <iconset resource="qProg_arm1986.qrc">
     <normaloff>:/new/prefix1/images/progtest.PNG</normaloff>:/new/prefix1/images/progtest.PNG</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_Connection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection          0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Connect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Ports">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Rescan">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Rescan ports</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Speed">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>62</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Verify">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Verify after program</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Run">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Run device</string>
   </property>
   <property name="icon">
    <iconset resource="qProg_arm1986.qrc">
     <normaloff>:/new/prefix1/images/progstart.PNG</normaloff>:/new/prefix1/images/progstart.PNG</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Run">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run after program</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qProg_arm1986.qrc"/>
 </resources>
 <connections/>
</ui>
